centinel/client.py

```diff
@@ -37,7 +37,7 @@ def get_input_file(self, experiment_name):
         return os.path.join(self.config['dirs']['data_dir'], input_file)
 
     def load_input_file(self, name):
-        input_file = self.get_input_file(name)
+        input_file = self.get_input_file("%s.txt" %(name))
```

Comment:
I made this change before I left Citizen Lab to make sure that input files could be used without extension constraints. However, I didn't make a new package with the client-side support. Therefore, I was running the server without this. If you make a new client package, this change will be included.